tests/playwright/upgrade-test/index.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-console */
+
+import { execSync } from 'child_process';
+export class UpgradeElementor {
+	cmd( cmd: string ) {
+		try {
+			console.log( cmd );
+			const text = execSync( cmd ).toString();
+			console.log( text );
+		} catch ( e ) {
+			console.error( e.toString() );
+		}
+	}
+
+	cleanUpBeforeTest() {
+		if ( ! process.env.CI ) {
+			this.cmd( 'rm -rf  plugin' );
+			this.cmd( 'rm -rf  ../../../elementor' );
+		}
+	}
+
+	getBuild() {
+		this.cmd( 'mkdir plugin' );
+		this.cmd( 'cd ../../../ && mkdir elementor' );
+		this.cmd( 'cd ../../../ && cp -r ./build/* elementor && zip -r elementor elementor' );
+		this.cmd( 'cd ../../../ && mv ./elementor.zip ./tests/playwright/upgrade-test/plugin' );
+	}
+
+	runServer() {
+		this.cmd( 'npx wp-env start' );
+	}
+
+	installLatestPluginFromWP() {
+		this.cmd( `npx wp-env run cli wp plugin install elementor --activate` );
```

Comment:
is there a way to select the plugin version?

Reply:
yes, `wp plugin install elementor --version=<version>`.

Reply:
added option to select plugin version. https://github.com/dmaziar/elementor/actions/runs/6421936744/job/17437459976

```
version is: 3.15.0
npx wp-env run cli bash -c 'wp plugin install elementor --version=3.15.0  --activate --force'
Installing Elementor Website Builder – More than Just a Page Builder (3.15.0)
```

Upgrade looks like this:
```
npx wp-env run cli wp plugin install ./plugin/elementor.zip --force
Unpacking the package...
Installing the plugin...
Removing the old version of the plugin...
Plugin updated successfully.
Success: Installed 1 of 1 plugins.

npx wp-env run cli wp plugin list
2023-10-05 16:46:44 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.15.0',
  'to' => '3.18.0',
)]
```